lib/WUI/CMakeLists.txt

```diff
@@ -31,10 +31,13 @@ target_sources(
             nhttp/status_page.cpp
             nhttp/upload_state.cpp
             random.cpp
+            random.cpp
+            sntp/sntp.c
             sntp/sntp.c
             sntp/sntp.c
             sntp/sntp_client.c
             sntp/sntp_client.c
+            sntp/sntp_client.c
```

Comment:
These are kind of plaguing the file, aren't they? How came we have `random` twice, and the `sntp` things _three times_? :-O

Reply:
Should be fixed now